### ISSUE TITLE
Make lastly opened side panel item persisted across the relaunch

### ISSUE THREAD

**Issue by simonhong** [brave/brave-browser#34399]
## Description 

After launch, `ctrl`(or `cmd`) + b  always opens reading list

## Steps to Reproduce

   1. Open bookmarks panel
   2. Relaunch
   3. `ctrl` + b

## Actual result:

Reading list panel is opened

## Expected result:
Lastly used bookmarks panel is opened

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

```
Brave | 1.62.74 Chromium: 119.0.6045.163 (Official Build) nightly (x86_64)
-- | --
Revision | 070cd801c60b65bed66cfed719aeab87384ec9b8
OS | macOS Version 14.2 (Build 23C5047e)
```

### Steps to Reproduce:
1. installed `1.62.74`
2. launched Brave
3. clicked on `Bookmarks` in the sidebar
4. confirmed the `Bookmarks` panel expanded open
5. shut down Brave
6. relaunched Brave
7. pressed `ctrl` + `b`

## Confirmed the Bookmarks panel was opened

**Comment by valynor** [comment]
Just tried this in 1.62.153 Chromium: 121.0.6167.85 (Official Build) (arm64) 
and it is NOT opening the last opened side panel (LEO) but Reading List.

**Comment by valynor** [comment]
Not fixed on my machine: 1.62.153 Chromium: 121.0.6167.85 (Official Build) (arm64) 

Last closed sidebar: LEO - restarted Brave and sidebar opens to Reading List, same as before 1.62.x
